Hogajama/hogajama-domain/src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence xmlns="http://java.sun.com/xml/ns/persistence" version="2.0">
+
+    <persistence-unit name="HOGARAMA">
+        <jta-data-source>java:jboss/datasources/Hogajama</jta-data-source>
+        <properties>
+            <property name="hibernate.show_sql" value="true"/>
+            <property name="hibernate.format_sql" value="true"/>
+            <property name="hibernate.hbm2ddl.auto" value="validate"/>
+            <property name="hibernate.default_schema" value="management" />
```

Comment:
Is there a reason you don't use the schema of the user, but a fixed schema "management"? This makes it harder to find the tables when logging into postgres with psql